src/applications/personalization/dashboard/components/Dashboard.jsx

```diff
@@ -151,11 +153,17 @@ const Dashboard = ({
     () => {
       if (isLOA3) {
         fetchFullName();
-        fetchMilitaryInformation();
-        fetchTotalDisabilityRating();
+        if (canAccessMilitaryHistory) {
+          fetchMilitaryInformation();
+        }
+        if (canAccessRatingInfo) {
+          fetchTotalDisabilityRating();
+        }
```

Comment:
I was wondering if we could somehow wrap our fetch calls in a Promise.allSettled or Promise.all to kick off whatever api calls are needed in parallel and speed up loading a litte. Not sure if it would help, but might be worth exploring in the future.